File: no_user_input_model/8_ball.py
import random


answer = (random.randrange(0, 10))

print(f'"MAGIC 8-BALL:')

if (answer == 0):
    print("It is certain.")
elif (answer == 1):
    print("As I see it, yes.")
elif (answer == 2):
    print("Without a doubt.")
elif (answer == 3):
    print("Outlook good.")
elif (answer == 4):
    print("Yes definitely.")
elif (answer == 5):
    print("Ask again later.")
elif (answer == 6):
    print("Cannot predict now.")
elif (answer == 7):
    print("Reply hazy, try again.")
elif (answer == 8):
    print("My reply is no.")
else:
    print("Very doubtful.")

File: no_user_input_model/8_ball.js
console.log("MAGIC 8-BALL:");

let answer = Math.floor(Math.random() * 10);

if (answer === 0) {
    console.log("It is certain.");
} else if (answer === 1) {
    console.log("As I see it, yes.");
} else if (answer === 2) {
    console.log("Without a doubt.");
} else if (answer === 3) {
    console.log("Outlook good.");
} else if (answer === 4) {
    console.log("Yes definitely.");
} else if (answer === 5) {
    console.log("Ask again later.");
} else if (answer === 6) {
    console.log("Cannot predict now.");
} else if (answer === 7) {
    console.log("Reply hazy, try again.");
} else if (answer === 8) {
    console.log("My reply is no.");
} else {
    console.log("Very doubtful.");
}
